doc/src/tutorials/physics/mechanics/duffing-example.rst

```diff
@@ -0,0 +1,141 @@
+
+   >>> B.set_ang_vel(N, q2.diff() * N.z)
+   >>> B.ang_vel_in(N)
+   Derivative(q2(t), t)*N.z
+
+Locations and velocities of the Duffing Oscillator block and the pendulum are:
+
+   >>> O = me.Point('O')
+   >>> Block = O.locatenew('Block', q1 * N.y)
+   >>> Pendulum = Block.locatenew('Pendulum', l * B.y)
+
+O is a fixed point in the inertial reference frame.
+
+   >>> O.set_vel(N, 0)
+   >>> Block.set_vel(N, q1d * N.y)
+   >>> Pendulum.v2pt_theory(Block, N, B)
+   Derivative(q1(t), t)*N.y - l*Derivative(q2(t), t)*B.x
+
+   >>> me.init_vprinting()
+
+Define inertia and rigid bodies.
+Here, we assume a simple pendulum which consists of a bob of mass m hanging from a massless string of length l
+and fixed at a pivot point (Duffing Oscillator Block).
+
+   >>> I_block = me.inertia(N, M*(h**2 + d**2)/12, M*(w**2 + h**2)/12, M*(w**2 + d**2)/12)
+   >>> I_pendulum = me.inertia(B, 0, 0, 2*m*r**2/5)
+
+   >>> par_block = me.RigidBody('block', Block, N, M, (I_block, Block))
+   >>> par_pendulum = me.RigidBody('pendulum', Pendulum, B, m, (I_pendulum, Pendulum))
+
+Define Forces
+=============
+
+We calculate the forces acting on the system.
+
+   >>> path = me.LinearPathway(O, Block)
+   >>> spring = me.DuffingSpring(k1, k2, path, 0)
+   >>> damper = me.LinearDamper(c1, path)
+
+   >>> loads = spring.to_loads() + damper.to_loads()
+
+   >>> bodies = [par_block, par_pendulum]
+
+   >>> for body in bodies:
+   ...     gravitational_force = -body.mass * g * N.y
+   ...     gravitational_load = (body.masscenter, gravitational_force)
+   ...     loads.append(gravitational_load)
+
+   >>> loads
+         /      _____           3/2\                  /        _____           3/2\
+         |     /   2       /  2\   |                  |       /   2       /  2\   |
+         \k1*\/  q1   + k2*\q1 /   /*q1               \- k1*\/  q1   - k2*\q1 /   /*q1
+    [(O, ------------------------------ n_y), (Block, -------------------------------- n_y), (O, c1*q1'(t) n_y), (Block, -c1*q1'(t) n_y), (Block, -M*g n_y), (Pendulum, -g*m n_y)]
+                       _____                                         _____
+                      /   2                                         /   2
+                    \/  q1                                        \/  q1
+
+Lagrange's Method
+=================
+
+With the problem setup, the Lagrangian can be calculated, and the equations of motion formed.
+
+   >>> L = me.Lagrangian(N, par_block, par_pendulum)
+   >>> me.Lagrangian(N, par_block, par_pendulum)
+            2      2       2     / 2       2                                     2\
+    M*q1'(t)    m*r *q2'(t)    m*\l *q2'(t)  - 2*l*sin(q2)*q1'(t)*q2'(t) + q1'(t) /
+    --------- + ------------ + ----------------------------------------------------
+        2            5                                  2
+
+   >>> LM = me.LagrangesMethod(L, [q1, q2], forcelist = loads, frame = N)
+   >>> LM.form_lagranges_equations()
```

Comment:
```suggestion
   >>> sm.simplify(LM.form_lagranges_equations())
```
Would recommend comparing results of `sm.simplify` and something like `sm.expand` to see which is more readable and looks closer to the EoMs shown in the paper (they are of course slightly different as you describe a slightly different system).